framework/test/unit/governance/data-lake-catalog.test.ts

```diff
@@ -35,6 +214,128 @@ describe('Create catalog for bronze, silver, gold with no global data removal',
     );
   });
 
+  test('should create crawlers that uses the databaseName', () => {
```

Comment:
Should test the database name contains `exampledb`

Reply:
Fixed